src/test/java/api/loans/anonymization/AnonymizeLoansImmediatelyAPITests.java

```diff
@@ -0,0 +1,224 @@
+package api.loans.anonymization;
+
+import static api.support.matchers.LoanMatchers.isAnonymized;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.junit.MatcherAssert.assertThat;
+
+import java.net.MalformedURLException;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import api.support.builders.CheckOutByBarcodeRequestBuilder;
+import api.support.builders.LoanHistoryConfigurationBuilder;
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeUtils;
+import org.junit.Test;
+
+public class AnonymizeLoansImmediatelyAPITests extends LoanAnonymizationTests {
+
+  /**
+   * Scenario 1
```

Comment:
Can we drop the scenario numbers? They have no context. Why are scenario 1.1 and 1.2 at the bottom of the file? It is just confusing.

Reply:
Sure we can. Done